RESOURCE.md

```diff
@@ -0,0 +1,48 @@
+# RESOURCES
+
+Stackdriver has defined [resource type](https://cloud.google.com/monitoring/api/resources) for monitoring and for each resource type there
+are mandatory resource labels. OpenCensus has defined [standard resource](https://github.com/census-instrumentation/opencensus-specs/blob/master/resource/StandardResources.md)
+types and labels. 
+This document describes the translation from OpenCensus resources to Stackdriver resources
+performed by this exporter.
+
+## Mapping between Stackdriver and OpenCensus Resources
+
+### k8s_container
```

Comment:
You can also mention `k8s_container` takes precedence, i.e GKE will be mapped to `k8s_container` and won't have GCP specific labels.

Reply:
done.